amazon_payments/api.py

```diff
@@ -193,7 +193,7 @@ def get_amazon_order_details(self, billing_agreement_id, access_token,
         return True, amazon_order_details
 
     def create_order_reference_id(self, billing_agreement_id, order_amount,
-                                  currency, **kwargs):
+                                  currency, order_id='N/A', **kwargs):
```

Comment:
the intent is that we may not have an actual order object at this point so the string must be constructed and passed here.